src/System.IO/pkg/aot/System.IO.pkgproj

```diff
@@ -7,9 +7,6 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\System.IO.csproj">
-      <TargetGroup>netstandard13aot</TargetGroup>
```

Comment:
What is your expectation for libraries that need to target CoreRT? What target group do you expect them to build with?